safe/test/data/hazard/volcano_krb.xml

```diff
@@ -232,6 +232,9 @@
           <layer_mode>
             <gco:CharacterString>classified</gco:CharacterString>
           </layer_mode>
+          <hazard_classification>
+            <gco:CharacterString>volcano_vector_hazard_classes</gco:CharacterString>
```

Comment:
Fixed in incoming commit. It was from bad copy and paste in volcano_hazard_classes definition.